external/ag-website-shared/src/components/features-section/DocsFeaturesSection.json

```diff
@@ -0,0 +1,162 @@
+                "description": "Additional UI elements for enhanced user interaction and customization.",
+                "link": "./context-menu"
+            }
+        ]
+    },
+    "charts": {
+        "community": [
+            {
+                "title": "Chart Essentials",
+                "description": "Basic chart types, including line, bar, column, and pie charts."
+            },
+            {
+                "title": "Accessibility Support",
+                "description": "Fully accessible charts with keyboard navigation and screen reader support.",
+                "link": "./accessibility/"
+            },
+            {
+                "title": "Basic Interactivity",
+                "description": "Highlight data items or entire series and add tooltips to provide extra contextual information.",
+                "link": "./series-highlighting/"
+            },
+            {
+                "title": "Intuitive Data Handling",
+                "description": "Handles your data in its existing format, no transformation required."
+            },
+            {
+                "title": "Themes and Styling",
+                "description": "Customize backgrounds, layouts, overlays, and themes to match your brand.",
+                "link": "./themes/"
+            },
+            {
+                "title": "Localization",
+                "description": "Support for 31 languages and integration with localization engines.",
+                "link": "./localisation/"
+            },
+            {
+                "title": "Download API",
+                "description": "Trigger browser-based image downloads of Charts in Base64 and PNG.",
+                "link": "./api-download/"
+            },
+            {
+                "title": "Event API",
+                "description": "Listen to and respond to user interactions with Charts and Series.",
+                "link": "./events/"
+            },
+            {
+                "title": "Major Frameworks",
+                "description": "Support for React, Angular and Vue, in addition to vanilla JavaScript."
+            }
+        ],
+        "enterprise": [
+            {
+                "title": "Advanced Chart Types",
+                "description": "Access to 20+ additional chart types, including Range, Heatmap, Sankey and more."
+            },
+            {
+                "title": "Financial Charts",
+                "description": "Build interactive financial charts with minimal configuration.",
+                "link": "./financial-charts/"
+            },
+            {
+                "title": "Animations",
+                "description": "Control charts initial load and data update animations.",
+                "link": "./animation/"
+            },
+            {
+                "title": "Advanced Interactivity",
+                "description": "Context menus, cross-hairs, navigator, zoom and chart synchronization."
+            },
```

Comment:
Link to Context menu from here? https://localhost:4600/charts/javascript/context-menu/